admin/RunningHAWQ.html.md.erb

```diff
@@ -2,21 +2,36 @@
 title: Running a HAWQ Cluster
 ---
 
-This section provides information for system administrators and database superusers responsible for administering a HAWQ system.
-
-This guide provides information and instructions for configuring, maintaining and using a HAWQ system. This guide is intended for system and database administrators responsible for managing a HAWQ system.
-
-This guide assumes knowledge of Linux/UNIX system administration, database management systems, database administration, and structured query language \(SQL\). Because HAWQ is based on PostgreSQL 8.2.15, this guide assumes some familiarity with PostgreSQL. This guide calls out similarities between HAWQ and PostgreSQL features throughout. It contains the topics:
-
-*  <a class="subnav" href="./ambari-admin.html">Managing HAWQ Using Ambari</a>
-*  <a class="subnav" href="./startstop.html">Starting and Stopping HAWQ</a>
-*  <a class="subnav" href="./ClusterExpansion.html">Expanding a Cluster</a>
-*  <a class="subnav" href="./ClusterShrink.html">Removing a Node</a>
-*  <a class="subnav" href="./BackingUpandRestoringHAWQDatabases.html">Backing Up and Restoring HAWQ</a>
-*  <a class="subnav" href="./HighAvailability.html">High Availability in HAWQ</a>
-*  <a class="subnav" href="./MasterMirroring.html">Master Mirroring</a>
-*  <a class="subnav" href="./HAWQFilespacesandHighAvailabilityEnabledHDFS.html">HAWQ Filespaces and High Availability Enabled HDFS</a>
-*  <a class="subnav" href="./FaultTolerance.html">Understanding the Fault Tolerance Service</a>
-*  <a class="subnav" href="./RecommendedMonitoringTasks.html">Recommended Monitoring and Maintenance Tasks</a>
-*  <a class="subnav" href="./maintain.html">Routine System Maintenance Tasks</a>
-*  <a class="subnav" href="./monitor.html">Monitoring a HAWQ System</a>
+Your HAWQ deployment environment is a collection of systems, processes, and data, and the users that operate on these systems, processes, and data. This section provides information for system administrators and database superusers responsible for administering a HAWQ deployment.
+
+You should have some knowledge of Linux/UNIX system administration, database management systems, database administration, and structured query language \(SQL\) to administer a HAWQ cluster. Because HAWQ is based on PostgreSQL 8.2.15, you should also have some familiarity with PostgreSQL. The HAWQ documentation calls out similarities between HAWQ and PostgreSQL features throughout.
+
```

Comment:
I don't think it's necessarily useful to call out that postgresql version, given that we've ported features from later versions as well.  It's probably enough to just say it's based on postgresql to give them a frame of reference.